sycl/test/warnings/sycl_2020_deprecations.cpp

```diff
@@ -1,7 +1,5 @@
-// RUN: %clangxx -fsycl -sycl-std=2020 -Xclang -verify -Xclang -verify-ignore-unexpected=note %s -o %t.out
-// RUN: %clangxx -fsycl -Xclang -verify -Xclang -verify-ignore-unexpected=note %s -o %t.out
-// RUN: %clangxx -fsycl -sycl-std=2017 -Werror %s -o %t.out
```

Comment:
Could you please comment on why do you remove the last two run lines? They check that no warnings are generated if the build is done for SYCL standards prior to SYCL 2020.

Reply:
That is my bad, missed `-sycl-std` flag there, returned them in 118ec382ffcb5d04ae1bf317e705024f875591bd

Reply:
> They check that no warnings are generated if the build is done for SYCL standards prior to SYCL 2020.

> returned them in [118ec38](https://github.com/intel/llvm/commit/118ec382ffcb5d04ae1bf317e705024f875591bd)

I don't think that "returned" checks do what Vladimir said i.e. "check that no warnings are generated if the build is done for SYCL standards prior to SYCL 2020". Right?